ts/Icon/index.tsx

```diff
@@ -14,7 +14,7 @@ export class Icon<PropsType = {}, StateType = {}> extends MaterialComponent<
 
   protected materialDom(props) {
     const classes = ['material-icons'];
-    // CardActionIcon sends className
+    // CardActionIcon sends className  TODO: Isn't that handled by MaterialComponent
```

Comment:
That is user given class.
This is talking about https://github.com/prateekbh/preact-material-components/blob/master/ts/Card/index.tsx#L88